framework/src/modules/interoperability/mainchain/store.ts

```diff
@@ -33,6 +44,78 @@ export class MainchainInteroperabilityStore extends BaseInteroperabilityStore {
 		return true;
 	}
 
+	public async forward(ccmForwardContext: CCMForwardContext): Promise<void> {
+		const {
+			ccm,
+			eventQueue,
+			logger,
+			networkIdentifier,
+			getAPIContext,
+			getStore,
+		} = ccmForwardContext;
+		const apiContext = getAPIContext();
+		const tokenAPI = this.interoperableModuleAPIs.get(MODULE_ID_TOKEN) as TokenAPI | undefined;
```

Comment:
You don't have to typecast to `| undefined`
```suggestion
		const tokenAPI = this.interoperableModuleAPIs.get(MODULE_ID_TOKEN) as TokenAPI;
```

Reply:
also, its `tokenCCAPI`, this separation is not their in Token LIP but we will make some APIs to be CCAPIs that are only callable from interoperability module